content/md/what-is/what-about-reference-entities.md

```diff
@@ -1,24 +1,25 @@
 ---
 id: what-about-reference-entities
 themes: first-steps, manage-your-reference-entities
-title: What about **reference entities**?
+title: What is a **reference entity**?
 popular: true
 ee-only: true
 related: manage-reference-entities, enrich-your-reference-entity-records
 ---
 
-# What's the use case?
+# Introduction to reference entities
 
-Let's take an example of a product page.  
+In order to present you quickly this attribute type, let's take an example of a product page.  
 ![Reference entity use case](../img/what-about-reference-entities.png)
 
-There are many types of product information such as text, images...  
-Some information are shared between different products (such as the definition of care instructions, colors or brands).  
-This shared information can be complex with their own attributes (e.g. a label, a logo, a description or photos for the brand).  
-They may have dedicated pages on the e-commerce website (e.g. dedicated pages for each brand) or their information may enrich each product page (e.g. logo of the brand).
+As you can see above are **many types of product information** visible on this page: text, images... 
+
+Some information are **shared between different products** (such as care instructions, or colors or even brands). This **data can be complex to manage because it has its own attributes** (e.g. a label, a logo, a description or photos).  
+Those information may have dedicated pages on one's e-commerce website (e.g. a webpage describing a brand) or their information may be used to enrich other product pages (e.g. the logo of a brand).
 
 # A bit of vocabulary
-## What's a reference entity?
+
+## Introduction to the reference entity
```

Comment:
Ca fait bizarre d'avoir 2 fois très proche " Introduction to reference entities" puis "Introduction to the reference entity" surtout qu'après on a "What is a record", je conserverai "What is a reference entity?" ici

Reply:
tu as raison, c'est modifié :)